posthog/models/filters/mixins/paths.py

```diff
@@ -31,6 +31,15 @@
 FunnelPathsType = Literal["funnel_path_before_step", "funnel_path_between_steps", "funnel_path_after_step"]
 
 
+def remove_trailing_slash(input: Optional[str]) -> Optional[str]:
+    if not input:
+        return input
+
+    if len(input) > 1 and input.endswith("/"):
+        return input[:-1]
+    return input
```

Comment:
Nit: I guess the first two `if`s could be just one? (`if input and ...`)

Reply:
the length of input needs to be greater than 1, since if its just '/', we don't want turn it into an empty string.

Reply:
Yeah, as in `if input is not None and len(input) > 1 and input.endswith("/")`

Reply:
Ahh gotcha!